litmus-portal/README.md

```diff
@@ -39,7 +39,6 @@ kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/lit
 curl https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/namespaced-K8s-template.yml --output litmus-portal-namespaced-K8s-template.yml
 envsubst < litmus-portal-namespaced-K8s-template.yml > ${LITMUS_PORTAL_NAMESPACE}-ns-scoped-litmus-portal-manifest.yml
 kubectl apply -f ${LITMUS_PORTAL_NAMESPACE}-ns-scoped-litmus-portal-manifest.yml -n ${LITMUS_PORTAL_NAMESPACE}
-kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/platforms/okteto/hello-world-AUT.yml -n ${LITMUS_PORTAL_NAMESPACE}
```

Comment:
This might be required @rajdas98 for running the chaos workflow specifically meant for Okteto, we have it on portal.

Reply:
can we move this to oketeto readme? These installation steps need to be generic for everyone

Reply:
But for namespaces scope unless someone installs this hello world app, no other predefined workflow will work @rajdas98 cc: @ksatchit